app/models/configuration_script_source.rb

```diff
@@ -1,5 +1,5 @@
 class ConfigurationScriptSource < ApplicationRecord
-  has_many    :configuration_script_payloads
+  has_many    :configuration_script_payloads, :dependent => :nullify
```

Comment:
I think this should be `:dependent => :destroy` since the playbook is an array off of the project in tower.